packages/python-mmh3/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgbase=python-mmh3
+pkgname=('python-mmh3')
+_pkgname=mmh3
+pkgver=2.5.1
+pkgrel=1
+pkgdesc='Python wrapper for MurmurHash (MurmurHash3), a set of fast and robust hash functions'
+arch=('any')
+url='https://pypi.org/project/mmh3/#files'
+license=('Creative')
```

Comment:
This does not exist in `/usr/share/licenses/common/` so check https://wiki.archlinux.org/index.php/PKGBUILD#license (eg. use custom)